docs/sources/administration/configuration.md

```diff
@@ -772,6 +772,22 @@ Refer to [Auth proxy authentication]({{< relref "../auth/auth-proxy.md" >}}) for
 
 Refer to [LDAP authentication]({{< relref "../auth/ldap.md" >}}) for detailed instructions.
 
+## [aws]
+
+You can configure core and external AWS plugins.
+
+### allowed_auth_providers
+
+Specify what authentication providers that the AWS plugins allow. For a list of allowed providers, refer to the data-source configuration page for a given plugin. If you configure a plugin by provisioning, only providers that are specified in `allowed_auth_providers` are allowed.
+
+Options: `default` (AWS SDK Default), `keys` (Access && key secret key), `credentials` (Credentials field), `ec2_IAM_role` (EC2 IAM Role)
+
+### assume_role_enabled
```

Comment:
Why is `_enabled` baked into this?

Reply:
The h3 headers in this doc maps to a config option in the [default.ini](https://github.com/grafana/grafana/blob/master/conf/defaults.ini) file. The reason it is suffixed with `_enabled` is because that's how all boolean options are named in this file, so I'm just following the used convention.